domain/ide/command/get_feature_flag_status.go

```diff
@@ -35,15 +38,25 @@ func (cmd *featureFlagStatus) Command() snyk.CommandData {
 
 func (cmd *featureFlagStatus) Execute(ctx context.Context) (any, error) {
 	if config.CurrentConfig().Token() == "" {
-		return nil, nil
+		return nil, errors.New("missing Snyk token")
 	}
 
 	args := cmd.command.Arguments
 	if len(args) < 1 {
-		return nil, nil
+		return nil, errors.New("missing feature flag required argument")
```

Comment:
nitpick: let's explain which arg is missing, e.g. `missing feature flag required argument "feature flag name"`